src/types/grid.graphql

```diff
@@ -0,0 +1,73 @@
+type Grid {
+  "The unique concept id assigned to the grid."
+  conceptId: String!
+  "The urls related to this grid"
+  relatedUrls: JSON
+  "The organization that this grid belongs to"
+  organization: JSON
+  "The contact mechanisms for this grid"
+  contactMechanisms: JSON
+  "The additional attribute to the grid."
+  additionalAttribute: JSON
+  "The description for this grid"
+  description: JSON
```

Comment:
This is a String value

Reply:
Ah I see why I got confused here.
`"Name": {
      "description": "Short name for this Grid, may not be human readable and must be unique for the provider.",
      "type": "string",
      "minLength": 1,
      "maxLength": 85
    },`
    I was using the schema.json instead of the example, so I thought many of these were JSON

Reply:
Converted to String